s3_to_redshift.go

```diff
@@ -26,6 +27,7 @@ var (
 	dataDate        = flag.String("date", "", "data date we should process, must be full RFC3339")
 	configFile      = flag.String("config", "", "schema & table config to use in YAML format")
 	gzip            = flag.Bool("gzip", true, "whether target files are gzipped, defaults to true")
+	timeGranularity = flag.String("granularity", "day", "how often we expect to append new data")
```

Comment:
I wonder if time granularity should be a required flag rather than defaulting to a day. Do what you want here, but I'd recommend thinking about it.